README.md

```diff
@@ -128,10 +134,6 @@ will include the flags `{isDuplicate: true, cause: 'digestCache'}`.
 **NOTE**: this is likely a temporary measure, if we start locking listeners to
```

Comment:
Is this still relevant?

Reply:
Yes, though I'm realizing my recent analytics-dynamodb-lambda changes may have impacted this.  Will open a ticket.

Reply:
PRX/analytics-ingest-lambda#84